OpenSim/Examples/ControllerExample/tugOfWar_model_ThelenOnly.osim

```diff
@@ -281,10 +281,6 @@
 				<Thelen2003Muscle name="muscle1">
 					<!--Flag indicating whether the force is disabled or not. Disabled means that the force is not active in subsequent dynamics realizations.-->
 					<isDisabled>false</isDisabled>
-					<!--Minimum allowed value for control signal. Used primarily when solving for control values.-->
```

Comment:
I can add these. The one issue I see is that if you change the `minimum_activation` in the model file, the `min_control` will still be the default `minimum_activation`.  Therefore, the user has to update both as it is now.